CHANGELOG.markdown

```diff
@@ -1,3 +1,7 @@
+5.7.0 [unreleased]
```

Comment:
This isn't a breaking change, so it doesn't require a major version bump. Change this to `5.6.2` instead.